sharktank/sharktank/types/sharding.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""Specifications describing how a tensor, ops, layers and blocks are
+sharded."""
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from . import Theta
```

Comment:
I think this needs to be `from .theta import Theta` or something like that. Within a package, you always want to import from the leaf vs a partially initalized parent.

Reply:
I removed as it was actually unused, but will look out for this in the future.